makefiles/pyaes.mk

```diff
@@ -0,0 +1,42 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += pyaes
```

Comment:
Debian calls this `python3-pyaes`, we should do the same.

Reply:
I **literally** asked YOU about this; I'll change all libraries to it. For the record: Python packages will (and should) have the ``python3-`` prefix